exonum-java-binding/core/rust/Cargo.toml

```diff
@@ -37,3 +37,10 @@ rpath = true
 
 [profile.release]
 rpath = true
+
+# FIXME: using git dependency until new Exonum with MerkleDB will be released
+[patch.crates-io]
+exonum = { git = "https://github.com/exonum/exonum", rev = "2868b8b" }
+exonum-testkit = { git = "https://github.com/exonum/exonum", rev = "2868b8b" }
+exonum-configuration = { git = "https://github.com/exonum/exonum", rev = "2868b8b" }
+exonum-time = { git = "https://github.com/exonum/exonum", rev = "2868b8b" }
```

Comment:
Probably it worth applying changes from #1079? It also has btc anchoring disabled but uses `exonum 0.12`.

Reply:
It can be done in master, I think. There are questions to the present 1079.